### DISCUSSION TITLE
How to get started with Beamer slides in MystJS?

### DISCUSSION THREAD

**Discussion by DavidPowell** [executablebooks/meta#935] [Q&A]
I have some existing material arranged as a JupyterBook, that I would like to rearrange and convert into slides.

The MystJS [documentation on generating a PDF](https://myst-tools.org/docs/mystjs/creating-pdf-documents) mentions the possibility of generating Beamer slides from Myst source file, but doesn't give any guidance on how to do that. 

The [JTex documentation](https://myst-tools.org/docs/jtex/create-a-beamer-template) seems to imply that a custom template needs to be created, but my random fumblings were unsuccessful, and didn't even produce a meaningful error message (it just threw me into a Xetex interactive prompt). Is there an example somewhere of such a custom template, and a Myst source file that uses this template to generate slides?

**Comment by rowanc1** [comment]
Hi @DavidPowell -- we had Beamer templates working a few months ago, but may have regressed in the last month as we have pushed harder on the scientific article side. I have documented a bit more here: https://github.com/executablebooks/mystjs/issues/192 I will aim to address at least the docs tomorrow morning, but hopefully get you a working solution!

**Reply by DavidPowell** [reply]
Thanks @rowanc1 - no need to rush to fix the code on my behalf, most likely I'll just go the lazy way and convert myst -> docx -> manually paste into powerpoint, especially given that Beamer output is such a bleeding-edge feature.

I do agree that a warning in the docs would be helpful.

**Reply by rowanc1** [reply]
Glad that workflow will work for you! Docx works well. :) I have tagged you on the other issue, and will provide a few more updates there as we move forward with this. 🚀
